06-kotlin-mqtt/.gitignore

```diff
@@ -0,0 +1,16 @@
+.gradle
+.kotlintest
+/build/
+/out
+
+# Ignore Gradle GUI config
+gradle-app.setting
+
+# Avoid ignoring Gradle wrapper jar file (.jar files are usually ignored)
+!gradle-wrapper.jar
```

Comment:
...I don't understand this either. Jars are not ignored, why there's an exception to a missing rule?
And why do we need to ignore jars, anyway?